model1.py

```diff
@@ -2,17 +2,18 @@
 # SPDX-License-Identifier: Apache-2.0
 
 """
-Top-down model
+Top-down Model
 ==============
 
 This script gives an example on how to use Pyro forecast module to backtest and
-make a submission for M5 accuracy competition.
+make a submission for M5 accuracy/uncertainty competition.
 
 Using the top-down approach in [1], we first construct a model to predict
 the aggregated sales across all items. Then we will distribute the aggregated
 prediction to each product based on its total sales during the last 28 days.
 
-The result is barely better than the best benchmark model ESX from the competition.
+The results are barely better than the best benchmark models from both accuracy
```

Comment:
do you mean "a little bit better"? "barely" tends to have a negative connotation

Reply:
oh, thanks!